.semaphore/semaphore-scheduled-builds.yml

```diff
@@ -702,6 +702,40 @@ blocks:
           - sudo journalctl > logs/journalctl.txt
           - artifact push job --expire-in 1d logs
 
+- name: 'OpenStack integration (Yoga)'
+  run:
+    when: "true or change_in(['/networking-calico/'])"
+  dependencies: ["Prerequisites"]
+  task:
+    agent:
+      machine:
+        type: e1-standard-4
+        os_image: ubuntu2004
+    prologue:
+      commands:
+      - cd networking-calico
+    jobs:
+      - name: 'Unit and FV tests (tox) on Yoga'
+        commands:
+          - ../.semaphore/run-and-monitor tox.log make tox-yoga
+      - name: 'Mainline ST (DevStack + Tempest) on Yoga'
+        commands:
+          - dpkg -l "python*" || true
+          - sudo find / -name "*wrapt*" || true
```

Comment:
Is this just for debugging? Looks like we remove the python3-wrapt package unconditionally.

Reply:
This, it was just for debugging, in case the previous line didn't find any packages, but wrapt was pre-installed in some different way.